packages/hardhat-verify/src/index.ts

```diff
@@ -39,27 +43,27 @@ import {
   UnexpectedNumberOfFilesError,
   VerificationAPIUnexpectedMessageError,
   ContractVerificationFailedError,
-} from "./errors";
+} from "./internal/errors";
 import {
   sleep,
   encodeArguments,
   getCompilerVersions,
   printSupportedNetworks,
   resolveConstructorArguments,
   resolveLibraries,
-} from "./utilities";
-import { Etherscan } from "./etherscan";
-import { Bytecode } from "./solc/bytecode";
+} from "./internal/utilities";
+import { Etherscan } from "./internal/etherscan";
+import { Bytecode } from "./internal/solc/bytecode";
 import {
   ContractInformation,
   ExtendedContractInformation,
   extractInferredContractInformation,
   extractMatchingContractInformation,
   getLibraryInformation,
   LibraryToAddress,
-} from "./solc/artifacts";
+} from "./internal/solc/artifacts";
 
-import "./type-extensions";
+import "./internal/type-extensions";
```

Comment:
I wonder if we should move the etherscan/sourcify subtasks to their own module, and here we just add `import "./internal/tasks/etherscan";` and `import ./internal/tasks/sourcify";`. It's not something we are doing anywhere else, and it would make code navigation harder in certain scenarios, but maybe it's worth it? I'm on the fence, so up to you if you want to do it.